NZSLDict.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
```

Comment:
The guiltiest approve every.
I can barely follow, but I didn't find anything obviously wrong.